cinema/views.py

```diff
@@ -1 +1,40 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+from cinema.serializers import (
+    GenreSerializer, ActorSerializer, CinemaHallSerializer, MovieSerializer,
+    MovieListSerializer, MovieSessionSerializer, MovieSessionDetailSerializer
```

Comment:
If importing more than two items, it is better to write them on separate lines. This improves readability and maintenance of the code. 

Example:

From taxi.models import ( 
	Driver,
	Manufacturer,
	Car
)
Read more about topic [here](https://peps.python.org/pep-0008/#imports)